src/app/header/titles.component.ts

```diff
@@ -1,17 +1,17 @@
-import { Component, OnDestroy, OnInit, Input } from "@angular/core";
+import { Component, OnDestroy, OnInit } from "@angular/core";
 import { TitlesService } from "./titles.service";
 import { Breadcrumb } from "./breadcrumb";
 import { Subscription } from "rxjs";
 import { LoggerFactory, LogLevel, Logger } from "diagnostics/logger";
+import { Router } from "@angular/router";
 
 @Component({
     selector: `titles`,
     template: `
 <div class="titles">
     <ul class="breadcrumbs">
         <li *ngFor="let crumb of breadcrumbs; index as i">
-            <span [ngClass]="{root: !i}" *ngIf="!(crumb.routerLink)">{{crumb.label}}</span>
-            <a *ngIf="crumb.routerLink" [routerLink]="crumb.routerLink">{{crumb.label}}</a>
+            <span [ngClass]="{root: !i}" [class.color-active]="canNavigate(crumb)" (click)="navigate(crumb)">{{crumb.label}}</span>
```

Comment:
I think you should add 'tabindex="0"' to make this focusable. Otherwise users can't navigate with keyboard.

Reply:
And add accessibility-focusable class as well to set the style when it is focused.